prisma/seed.ts

```diff
@@ -0,0 +1,114 @@
+import { PrismaClient } from '@prisma/client';
+import * as bcrypt from 'bcrypt';
+
```

Comment:
Use `readFileSync` to read the SSH keyfile

```suggestion
import { readFileSync } from 'fs';

```